src/share/util/shr_flux_mod.F90

```diff
@@ -843,7 +849,7 @@ real(R8)    :: psix2   ! stability function at ztref reference height
             Hb = (Qdel/rcpocn)+(Fd*betaS/alphaT)
             Hb = min(Hb , 0.0_R8)
             lambdaV = lambdaC*(1.0_R8 + ( (0.0_R8-Hb)*16.0_R8*molvisc(tBulk(n))* &
-                 shr_const_g*alphaT*molPr(tBulk(n))**2/ustarw**4)**0.75)**(-1/3)
+                 shr_const_g*alphaT*molPr(tBulk(n))**2/ustarw**4)**0.75_R8)**(-1.0_R8/3.0_R8)
```

Comment:
These are in the diurnal code which is not currently exercised, do we really need another pr?